tests/lookup/tests.py

```diff
@@ -1855,6 +1855,14 @@ def test_in_lookup_in_filter(self):
                     Season.objects.filter(In(F("year"), years)).order_by("pk"), seasons
                 )
 
+    def test_in_lookup_in_filter_text_field(self):
+        self.assertSequenceEqual(
+            Season.objects.filter(
+                In(F("nulled_text_field"), [F("nulled_text_field"), "special_value"])
+            ),
+            [self.s2],
+        )
```

Comment:
This test crashes on Oracle because blob fields (e.g. `NCLOB`) are not allowed in the `IN()` clause. As far as I'm aware `TextFields` are unnecessary here, see #20453.